Tasks/ANTV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -39,6 +39,7 @@
   "loc.input.label.jdkArchitecture": "JDK architecture",
   "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of the JDK.",
   "loc.messages.LocateJVMBasedOnVersionAndArch": "Locate JAVA_HOME for Java %s %s",
+  "loc.messages.UnsupportedJdkWarning": "JDK 9 and JDK 10 are out of support, please switch to JDK 11 in your project and task. Attempting to build with JDK11 ...",
```

Comment:
"Out of support" -> "No longer supported" ?

Reply:
@DavidStaheli approved "out of support" :).